microsoft/testsuites/vm_extensions/common.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
The file name may be confused, because we asked all extensions test cases to be here. Can you put all 3 files into a subfolder?

Reply:
Agree that the 3 files should be in a subdirectory. Some of the code in this file seems specific to RunCommand (e,g, the check for architecture, because RunCommand is not published on ARM64 yet).